cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -309,9 +328,11 @@ func newCmdCertsExpiration(out io.Writer, kdir string) *cobra.Command {
 type expirationFlags struct {
 	cfgPath string
 	cfg     kubeadmapiv1beta2.ClusterConfiguration
+	withCA  bool
 }
 
 func addExpirationFlags(cmd *cobra.Command, flags *expirationFlags) {
 	options.AddConfigFlag(cmd.Flags(), &flags.cfgPath)
 	options.AddCertificateDirFlag(cmd.Flags(), &flags.cfg.CertificatesDir)
+	cmd.Flags().BoolVar(&flags.withCA, "with-ca", flags.withCA, "Show CA expiration information")
```

Comment:
i was under the impression that we decided to always print the CA expiration, but in a separate table.
i don't think this flag is needed, but maybe it was decided to have it and i missed this part of the conversation.

Reply:
Thanks @neolit123 
This is not really discussed.
This flag is added to account for the fact that there may be only a few cases where users are concerned about CA expiration information.
WDYT? @fabriziopandini @rosti

Reply:
we really should be adding flags only when absolutely needed and in the meantime think of ways how to deprecate / remove some of our existing flags.